docs/README.md

```diff
@@ -4,42 +4,48 @@
 
-Some x86 internal opcodes use parts of "Realmode X86 Emulator Library", see [x86primop.c](../src/emu/x86primop.c) for copyright details
+Logo and Icon made by @grayduck, thanks!
 
-Here's  6 videos, the first 2 videos are videos of "Airline Tycoon Deluxe" and "Heretic 2" running on a GigaHertz OpenPandora (the second one is using  the dynarec), and the next 2 videos are videos of of "Bit.Trip.Runner" and "Neverwinter Night" running on an ODroid XU4 (without dynarec), and the last 2  videos are on on a Pi4: Shovel Knight (video from @ITotalJustice) and Freedom Planet (video from @djazz), also without dynarec.
 
-[![Play on Youtube](https://img.youtube.com/vi/bLt0hMoFDLk/3.jpg)](https://www.youtube.com/watch?v=bLt0hMoFDLk) [![Play on Youtube](https://img.youtube.com/vi/MM7kWYts7IA/3.jpg)](https://www.youtube.com/watch?v=MM7kWYts7IA) [![Play on Youtube](https://img.youtube.com/vi/8hr71S029Hg/1.jpg)](https://www.youtube.com/watch?v=8hr71S029Hg) [![Play on Youtube](https://img.youtube.com/vi/B4YN37z3-ws/1.jpg)](https://www.youtube.com/watch?v=B4YN37z3-ws) [![Play on Youtube](https://img.youtube.com/vi/xk8Q30mxqPg/1.jpg)](https://www.youtube.com/watch?v=xk8Q30mxqPg) [![Play on Youtube](https://img.youtube.com/vi/_QMRMVvYrqU/1.jpg)](https://www.youtube.com/watch?v=_QMRMVvYrqU)
+Compiling/Installing
+----
+> Instructions to compile Box86 can be found [here](COMPILE.md)
+> 
+> Instructions to install Wine and winetricks for use with Box86 are [here](X86WINE.md)
+> 
+> For an easy 1-click installation of Box86, Wine, & winetricks on a Raspberry Pi, check out box86 on [pi-apps](https://github.com/Botspot/pi-apps/)
 
-You can find many more Box86 video on the [Pi Labs Youtube Channel](https://www.youtube.com/channel/UCgfQjdc5RceRlTGfuthBs7g) [![Pi Labs Youtube Channel](https://yt3.ggpht.com/a/AATXAJyMeWrgCjs78gr6To6yX4KtDPUCS7hsbX1rRA=s100-c-k-c0xffffffff-no-rj-mo)](https://www.youtube.com/channel/UCgfQjdc5RceRlTGfuthBs7g).
+You *NEED* a 32-bit subsystem to run and build Box86. Box86 is useless on 64-bit only systems. Also, you *NEED* a 32-bit toolchain to build Box86. A toolchain that only supports 64-bit will not compile Box86, and you'll get errors (typically on aarch64, you get "-marm" not recognized, and you'll need a multiarch or chroot environnement).
 
-Compatibility list is there: https://github.com/ptitSeb/box86-compatibility-list/issues
+If you are serious about developing Box86, you should install ccache and build Box86 with it. (Use ccmake for example.)
 
-<img src="img/Box86Icon.png" width="96" height="96">
+To enable TRACE (i.e. dumping to stdout all individual x86 instructions executed, with dump of registers), you'll also need [Zydis library](https://github.com/zyantific/zydis) available on your system.
 
-Logo and Icon made by @grayduck, thanks!
 
-Note that this project is not to be mistaken with [86box](https://github.com/86Box/86Box), a nice "Full system" emulator specialized in early (to fairly recent) PC hardware.
```

Comment:
Were is that part?

Reply:
Oh yeah, I moved it farther up.  I can move it back down to the bottom for visibility too.  No pressure on these edits too :)